embedded-mongodb/src/main/java/com/playtika/test/mongodb/MongodbStatusCheck.java

```diff
@@ -1,11 +1,23 @@
 package com.playtika.test.mongodb;
 
 import com.playtika.test.common.checks.AbstractCommandWaitStrategy;
+import lombok.RequiredArgsConstructor;
 
+@RequiredArgsConstructor
 public class MongodbStatusCheck extends AbstractCommandWaitStrategy {
+    private final MongodbProperties properties;
 
     @Override
     public String[] getCheckCommand() {
-        return new String[]{"mongo", "admin", "--eval", "\"db['system.version'].find()\""};
+        String shell = properties.getShell();
+        if (shell == null) {
```

Comment:
how it can be null?

Reply:
I based my changes on the current solution
https://github.com/PlaytikaOSS/testcontainers-spring-boot/blob/develop/embedded-mongodb/src/main/java/com/playtika/testcontainer/mongodb/MongodbStatusCheck.java#L12-L16
and yes, it looks redundant I will change it